README.md

```diff
@@ -92,43 +92,84 @@ The [deploy script](./deploy) does a similar thing but the end result is output
   ```bash
-  docker run -e CHANGELOG_GITHUB_TOKEN="$CHANGELOG_GITHUB_TOKEN" -it --rm -v "$(pwd)":/usr/local/src/your-app ferrarimarco/github-changelog-generator --user inviqa --project harness-base-php --exclude-labels "duplicate,question,invalid,wontfix,skip-changelog" --future-release <nextReleaseTag>
+  docker run -e CHANGELOG_GITHUB_TOKEN="$CHANGELOG_GITHUB_TOKEN" -it --rm -v "$(pwd)":/usr/local/src/your-app ferrarimarco/github-changelog-generator --user inviqa --project harness-base-php --exclude-labels "duplicate,question,invalid,wontfix,skip-changelog" --future-release 0.10.0
   ```
 4. Examine the generated CHANGELOG.md. For every entry in the `Merged pull requests` section, examine the Pull Requests
    and assign each pull request either a `enhancement` label for a new feature, `bug` for a bugfix or `deprecated` for
    a deprecation.
 5. For each Pull Request in the release, assign an appropriate `harness-*` label.
 6. Re-generate the changelog using step 3 as needed.
-7. Commit the resulting CHANGELOG.md, push and raise a pull request.
-8. Once merged, continue with the release process below.
+7. Adjust the version for each framework's README.md: `sed -i'' s/v0\.9\.0/v0.10.0/ src/*/README.md`
+8. Commit the resulting changes, push and raise a pull request.
+9. Once merged, continue with the release process below.
 
 ### Performing a Release
 
 Once the CHANGELOG.markdown is in the branch you wish to release:
 
-1. Tag the release version with `git tag <releaseVersion>`
-2. Push the tag to the repository: `git push origin <releaseVersion>`
+1. Tag the release version with `git tag 0.10.0 -m "v0.10.0"`
+2. Push the tag to the repository: `git push origin 0.10.0`
 3. Verify you don't have any ignored files in `src/`, and clean up if you do: `git status --ignored`
 4. Run the deploy script: `./deploy`
-5. On each downstream repository:
-   1. Tag the latest `0.9.x` branch push as the release version.
-   2. Push the tag to the repository
+5. Tag each downstream repository with the version being released: `./tag 0.10.0`
 6. Submit a pull request to [my127/my127.io] which adds the new release version and asset download URL for the
    php-based harnesses to `harnesses.json`
+7. Create a "Github Release" for this repository and downstream repositories, pasting in the changelog for the release:
+   - https://github.com/inviqa/harness-base-php/releases/new?tag=0.10.0
+   - https://github.com/inviqa/harness-php/releases/new?tag=0.10.0
+   - https://github.com/inviqa/harness-akeneo/releases/new?tag=0.10.0
+   - https://github.com/inviqa/harness-drupal8/releases/new?tag=0.10.0
+   - https://github.com/inviqa/harness-magento1/releases/new?tag=0.10.0
+   - https://github.com/inviqa/harness-magento2/releases/new?tag=0.10.0
+   - https://github.com/inviqa/harness-spryker/releases/new?tag=0.10.0
+   - https://github.com/inviqa/harness-symfony/releases/new?tag=0.10.0
+   - https://github.com/inviqa/harness-wordpress/releases/new?tag=0.10.0
```

Comment:
You can actually create the tag from the new release page

Reply:
Less typing if we push and then follow links.

Reply:
you still need to type to create a tag and push it, more so than following a link that can create a tag

Reply:
https://github.com/inviqa/harness-wordpress/releases/new?tag=0.10.0&target=0.10.x

Reply:
That's what the tag script does below, but the link you've provided is far better. Couldn't find that documented at github.

Will adjust.